src/app/home/home.component.ts

```diff
@@ -66,8 +68,14 @@ export class HomeComponent extends RouterEnter {
     this.loadingTodoItems = true;
     this.loadingProgress = true;
     this.loadingActivity = true;
+    this.achievements= [];
     // add a flag in local storage to indicate that is there any fast feedback open
     this.storage.set('fastFeedbackOpening', false);
+
+    this.achievementService.getAchievements('desc').subscribe(achievements => {
+      this.achievements = achievements;
+      this.achievements.length = 3;
```

Comment:
This is not enough. We need the logic to display certain 3 achievements. (e.g. 2 earned, 1 not earned, these logics)